server/jsonify.py

```diff
@@ -0,0 +1,68 @@
+###############################################################################
```

Comment:
I would propose to put the file to a folder and mentioned its purpose and usage in some readme or other documentation.

Reply:
Yes. thanks!